### DISCUSSION TITLE
Preliminary moderation

### DISCUSSION THREAD

**Discussion by dinozavrix** [forem/forem#16859] [Feature Requests]
Good evening, we are from Ukraine. We want to use your great platform to help developers across the country join forces to help our people cope with the war and its aftermath. We are afraid that bots from the Russian Federation can attack such a resource and ask you to consider the option of implementing barrier functions so that in the event of a bot attack, articles and comments can be published only after preliminary moderation and approval. Thank you 

Also about this request https://forem.dev/dinozavrix/preliminary-moderation-2e9m

**Comment by ildi-x** [comment]
At the moment, one suggestion that would work well if you wanted strong control over who can join and make posts/comments in your community, is to configure your Forem as an **invite only** community. You can do this by going to:

 _Admin > Customization > Authentication > Invite-only mode_
 
Your content is still public (anyone in the world is able to read) but only those who you invite via email can join and participate. Also @ellativity suggested to configure "**Rate limits and anti-spam**" which I think can also help you out.
